modules/creating-nfd-cr-cli.adoc

```diff
@@ -110,13 +104,11 @@ spec:
             - loadedKMod: ["example_kmod3"]
 ----
 
-For more details on how to customize NFD workers, refer to the link:https://kubernetes-sigs.github.io/node-feature-discovery/v0.10/advanced/worker-configuration-reference.html[Configuration file reference of nfd-worker].
-
-. Create the `NodeFeatureDiscovery` CR instance by running the following command:
+. Create the `NodeFeatureDiscovery` CR by running the following command:
 +
 [source,terminal]
 ----
-$ oc create -f NodeFeatureDiscovery.yaml
+$ oc apply -f <filename>
 ----
 
 .Verification
```

Comment:
On line 116, revise the wording to "To verify that the instance is created, run the following command:"